src/illustrativeIcons/icon-illustration-family.svg

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="80px" height="80px" viewBox="0 0 80 80" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
The sizes do not seem to be uniform. I think this should be reported to PAMU even if it does not make a difference in our case.

Reply:
This can also be left as maintenance task for later.

Reply:
Reported to PAMU.